### PR TITLE
Ignoring build files in Git.

### PR DESCRIPTION
**Pull request by trtikm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,8 @@ CMakeCache.txt
 
 *.ninja
 .ninja*
+install_manifest.txt
+googletest
+llvm-*/
+lib/symbioticpy/symbiotic/versions.py
```
Comment (lzaoral) [inline .gitignore:27]: Could you split this change into a separate commit please? So that it's easier to revert this change if we would ever had a reason to do that.

File: README.md [hunk 2]
```diff
@@ -37,10 +37,15 @@ First of all you must clone the repository:
 $ git clone https://github.com/staticafi/symbiotic
 ```
 
+Next you need to update submodules:
+```
+$ cd symbiotic
+$ git submodule update --init
```
Comment (lzaoral) [inline README.md:43]: What is the reason for such change? AFAIK it should not be necessary.

The submodules are initialised both in `build.sh` and `system-build.sh`: https://github.com/staticafi/symbiotic/blob/0534e840b4c778b343831ee69bcc7656fc2c3d28/system-build.sh#L318 https://github.com/staticafi/symbiotic/blob/0534e840b4c778b343831ee69bcc7656fc2c3d28/build.sh#L544

**Review by lzaoral — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (trtikm) [reply README.md:43]: I see. I'll revert this change. (Note: I think, it is unusual to build the project to obtain all of its sources).

File: .gitignore (hunk 1 above)
Comment (trtikm) [reply .gitignore:27]: Since I'll drop changes in the README file, this change will be separated.

**Review by lzaoral — APPROVED** [review]
LGTM! Thanks @trtikm!

**Comment by lzaoral** [comment]
@trtikm, could you please do fast-forward merges without a merge commit when possible so that we do not unnecessarily pollute the git history? Thanks you!
